src/canopy_init.F90

```diff
@@ -43,6 +43,18 @@ SUBROUTINE canopy_init
     if(allocated(ppfd_shade240_tmp))  ppfd_shade240_tmp(:,:,:)   = 0.0_rk
     if(allocated(tmp2mref_tmp))       tmp2mref_tmp(:,:)          = 0.0_rk
     if(allocated(ubzref_tmp))         ubzref_tmp(:,:)            = 0.0_rk
+    if(allocated(tleaf_sun24_rst))    tleaf_sun24_rst(:,:,:)     = fillreal
```

Comment:
Similar comment here and below for having new "rst" variables. Are these really needed>?